packages/host/app/components/card-prerender.gts

```diff
@@ -107,7 +107,6 @@ export default class CardPrerender extends Component {
         indexer,
         renderCard: this.renderService.renderCard.bind(this.renderService),
       }),
-      boom,
```

Comment:
Followup: it looks like this is from #1307

Reply:
whoops--I must have forgotten to switch branches when I made this change